OrbitCore/SamplingProfiler.h

```diff
@@ -137,6 +139,8 @@ class SamplingProfiler {
   void SortByThreadUsage();
   void ProcessSamples();
   void UpdateAddressInfo(uint64_t address);
+  ThreadSampleData& GetSummary() { return m_ThreadSampleData[0]; }
```

Comment:
GetSummary should probably be const or have a const-overload which returns a const ThreadSampleData&

Reply:
Yes it should! It was there before (just recently deleted). I will change it to a `GetSummary*` as it might be that the case that there is non yet.